code/model/SiteTree.php

```diff
@@ -191,7 +190,7 @@ class SiteTree extends DataObject implements PermissionProvider,i18nEntityProvid
 	
 	private static $searchable_fields = array(
 		'Title',
-		'Content',
+		'Content'
```

Comment:
Do you want that fixed to approve the PR, or just a general comment for whoever approves it?